src/codegen/wasm.zig

```diff
@@ -661,12 +668,12 @@ pub const Context = struct {
             .dbg_stmt => WValue.none,
             .load => self.genLoad(inst.castTag(.load).?),
             .loop => self.genLoop(inst.castTag(.loop).?),
-            .mul => self.genMul(inst.castTag(.mul).?),
+            .mul => self.genBinOp(inst.castTag(.mul).?, .mul),
             .not => self.genNot(inst.castTag(.not).?),
             .ret => self.genRet(inst.castTag(.ret).?),
             .retvoid => WValue.none,
             .store => self.genStore(inst.castTag(.store).?),
-            .sub => self.genSub(inst.castTag(.sub).?),
+            .sub => self.genBinOp(inst.castTag(.sub).?, .sub),
```

Comment:
There's room here for another refactor with another helper:
```
fn binOpTagToOp(tag: Inst.Tag) Op {
    return switch (tag) {
        .cmp_eq => .eq,
        .cmp_gte => .ge,
        ...
        .add => .add,
        ...
    };
}
``` but it's lower priority.